tests/libvmi/pod.go

```diff
@@ -0,0 +1,35 @@
+package libvmi
+
+import (
+	"fmt"
+
+	k8sv1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	v1 "kubevirt.io/client-go/api/v1"
+	"kubevirt.io/client-go/kubecli"
+	"kubevirt.io/kubevirt/pkg/controller"
+	testerrors "kubevirt.io/kubevirt/tests/errors"
+)
+
+func GetPodByVirtualMachineInstance(vmi *v1.VirtualMachineInstance, namespace string) *k8sv1.Pod {
```

Comment:
Moving it to status.go and ensuring that all the tests using the `libvmi.` version, I prefer not to create new commit for this it will cause too much pain.